### DISCUSSION TITLE
Time intelligence not loading into my Power BI

### DISCUSSION THREAD

**Discussion by TenaSheets2** [sql-bi/Bravo#786] [Q&A]
I have downloaded Bravo and can see the date data in my model but no time intelligence data.  It did not load "Time Intelligence" into the model.

![Time](https://github.com/sql-bi/Bravo/assets/167650678/d868b753-deab-4be5-9d56-99442ad60feb)

![tables](https://github.com/sql-bi/Bravo/assets/167650678/09956b68-0f39-44e7-9ad6-e55282347e30)

**Comment by albertospelta** [comment]
@TenaSheets2 Did the measures appear after saving and reopening the report in PBIDesktop? If so, it suggests a known issue (https://github.com/sql-bi/Bravo/issues/179#issuecomment-1075284256). We're aware that Power BI Desktop may not update when changes are made using external tools like Bravo or Tabular Editor. As of today, the only workaround seems to be forcing Power BI Desktop to synchronize, for example, by creating a new measure. While this is just a workaround, it should resolve the issue temporarily.
